app/actions/views/account_notifications.js

```diff
@@ -1,36 +0,0 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
-// See LICENSE.txt for license information.
-
-import {updateMe} from 'mattermost-redux/actions/users';
-import {Preferences} from 'mattermost-redux/constants';
-import {savePreferences} from 'mattermost-redux/actions/preferences';
-import {getConfig} from 'mattermost-redux/selectors/entities/general';
-
-export function handleUpdateUserNotifyProps(notifyProps) {
```

Comment:
This was deleted in favor of directly using `updateMe` redux action. I think this was put in place before to consider saving email interval to user's preference.  Since we are saving that preference on its own https://github.com/mattermost/mattermost-mobile/pull/2051/files#diff-137090d1468bb417ab6868ee91a4665eR106, this is no longer necessary.